ChromecastSampleApp/app/src/main/java/com/ooyala/sample/ChromecastListActivity.java

```diff
@@ -44,10 +45,14 @@ public void onCreate(Bundle savedInstanceState) {
   @Override
   public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
     final Intent intent = new Intent(this, videoList[position].activity);
-    intent.putExtra("embedcode", videoList[position].embedCode);
-    intent.putExtra("secondEmbedCode", videoList[position].secondEmbedCode);
-    intent.putExtra("pcode", videoList[position].pcode);
-    intent.putExtra("domain", videoList[position].domain);
+    SharedPreferences lastChoosenParams = getSharedPreferences("LastChoosenParams", MODE_PRIVATE);
```

Comment:
Why we need use SharedPrefernces for it??

Reply:
Because we have to save latest params for case when user open ChromecastPlayerActivity via minicontroller (we cannot paste intent with bundles)